tests/phpunit/tests/functions/wpRefererFeld.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
This code needs to be moved to the existing `wpRefererField.php` file :)

Reply:
Refreshed the branch and merged the tests